src/ev3_bluetooth.c

```diff
@@ -0,0 +1,332 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <poll.h>
+#include <ctype.h>
+#include "../firmware_headers/bluetooth/bluetooth.h"
```

Comment:
You can add the root directory to the search path, so the `..` becomes unncessary.

Reply:
Do you suggest to add to the search path the `firmware_headers` folder, so that the import looks like this:
```
#include "bluetooth/bluetooth.h"
```
Or to add the entire root directory, so that it looks like this?
```suggestion
#include "firmware_headers/bluetooth/bluetooth.h"
```

Because for the second scenario I think we would need to change all the imports, for example
```
#include "ev3_timer.h"
```
would become 
```suggestion
#include "API/ev3_timer.h"
```

(In the second case then I think that `API` should remain `API`, instead of becoming `src`, what do you think?)

Reply:
> Or to add the entire root directory, so that it looks like this?

This.

> Because for the second scenario I think we would need to change all the imports, for example

Nope this wouldn't be necessary. You can have multiple directories in the search path.

> Because for the second scenario I think we would need to change all the imports, for example

leaving `API` as `API` is probably for the best. Then I don't need to change the Windows Installer and Eclipse Plugin... ^^'

Reply:
Kept API. I didn't notice that the Makefile already included the root directory